app/models/bookings/booking.rb

```diff
@@ -0,0 +1,47 @@
+module Bookings
+  class Booking < ApplicationRecord
+    belongs_to :bookings_placement_request,
+      class_name: 'Bookings::PlacementRequest',
+      dependent: :destroy
```

Comment:
Not sure if we want this dependent destroy. Probably want to keep the placement request around if the school deletes the booking

Reply:
Wouldn't this mean that if the placement request was deleted the booking would be too?